code/datums/ruins/space.dm

```diff
@@ -356,3 +356,9 @@
 	suffix = "the_faceoff.dmm"
 	name = "The Faceoff"
 	description = "What do you get when a meeting of the enemy corporations get crashed?"
+
+/datum/map_template/ruin/space/the_faceoff
```

Comment:
i think you copypasted a bit too hard, make sure its a unique subtype

Reply:
i am so stupid